ci-operator/step-registry/rosa/cluster/index/rosa-cluster-index-commands.sh

```diff
@@ -67,15 +67,22 @@ index_metadata(){
   fi
 }
 
+calculate_total_install(){
```

Comment:
Isn't this same as the value being populated in this [panel](https://grafana.rdu2.scalelab.redhat.com:3000/d/ecd55f61-ab4a-44c2-a44a-8468125617a3/cpt-rosa-timers?orgId=1&var-datasource=e7252c28-8a7d-4a1e-9f65-18fec6eac3be&var-uuid=0600bd49-477a-4e47-bb18-bbd95918e0ed&var-major_version=All&var-version=All&var-group_by=uuid.keyword&var-nodes=24&var-Filters=hypershift%7C%3D%7Cfalse&from=1720046983584&to=1720651783584&viewPanel=55) here?

Reply:
@morenod ^^

Reply:
No, cluster ready timer do not include the nodes